tapdb/assets_store.go

```diff
@@ -3062,7 +3062,7 @@ func (a *AssetStore) LogAnchorTxConfirm(ctx context.Context,
 
 	var (
 		writeTxOpts    AssetStoreTxOptions
-		localProofKeys []asset.SerializedKey
+		localProofKeys []tapfreighter.OutputIdentifier
```

Comment:
> We recently changed the semantics around transfers. We now allow
> multiple virtual packets to be combined in a single transfer.

Wasn't this always the case if a user used the vPSBT APIs?

Reply:
I guess it was always the case **since** we implemented committing multiple vPackets to a single on-chain transaction, yes. Which was a while back, so perhaps "we recently changed" isn't super accurate.